canonical-web-and-design/ubuntu.com
# Remove /ai/install-kubeflow page and redirect to charmed-kubeflow.io/docs/install instead

## Done

> Would it be possible to point https://ubuntu.com/ai/install-kubeflow to https://charmed-kubeflow.io/docs/install ? 

> This second page is much more up to date, independent of Kubernetes, and also mentions MicroK8s as a quick path.

- Remove /ai/install-kubeflow and link to charmed-kubeflow.io in the redirects, removed from the subnav, updated older redirects

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/ai/install-kubeflow
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to http://0.0.0.0:8001/ai and see that 'Install Kubeflow' isn't in the sub menu or the mega menus
